src/ui/components/FilterSwitch.tsx

```diff
@@ -24,7 +24,7 @@ const TOGGLE_WIDTH = getSpacing(7)
 const TOGGLE_PATH_START = 2
 const TOGGLE_PATH_END = TOGGLE_WIDTH - TOGGLE_PATH_START
 
-const FilterSwitch: React.FC<Props> = (props: Props) => {
+const FilterSwitch: React.FC<FilterSwitchProps> = (props: FilterSwitchProps) => {
```

Comment:
il n'y a pas besoin de répéter le type des arguments lorsque la fonction la fonction est typée dans son ensemble

```suggestion
const FilterSwitch: React.FC<FilterSwitchProps> = (props) => {
```

`FC` est déprécié


```suggestion
const FilterSwitch: FunctionComponent<FilterSwitchProps> = (props) => {
```